onepager.py

```diff
@@ -55,31 +55,29 @@
 
 
 # Strings (str)
-standard_string = "string"
-standard_string = 'string'
```

Comment:
**issue**: so, the thinking around this was to show that you can have a string with double quote `"` or single quote `'`, which is an important distinction because in a language like C you cannot do this. Beyond that, you can even have triple quote `"""` for single or multi-line strings. It's important to show this information, so you might want to demonstrate it using different naming or with comment to the side.

Reply:
This might have been inadvertent when I was changing it to std_string to save space for the Encoding/Decoding section below, or I might have thought it was redundant since there are other examples of single and double quotes below it. Anyway, I agree it's important to show so I added further examples.